tools/npZ2ids.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+# npz2ids - tool for extracting object_ids from 3dscenegraph dataset
+#           (https://3dscenegraph.stanford.edu/)
+
+import sys
+
+import numpy as np
```

Comment:
Please use `argparse` to do the argument parsing https://docs.python.org/3/library/argparse.html